.github/workflows/tests.yaml

```diff
@@ -94,32 +94,34 @@ jobs:
         test: ${{ fromJSON(needs.setup-integration-tests.outputs.test_names) }}
     runs-on: ubuntu-latest
     steps:
-    # This step is needed to avoid running the integration tests if the secrets are not available.
-    # TODO: remove this step once we have a way to run integration tests on forks.
-    # https://github.com/Kong/kubernetes-testing-framework/issues/596
-    - name: Detect if we should run (have required secrets)
-      id: detect_if_should_run
+    # This step is needed to avoid running the integration tests requiring an enterprise license
+    # if the secrets are not available.
+    - name: Detect if we should run test cases requring an enterprise license (have required secrets)
+      id: detect_if_should_run_enterprise
       run: echo "result=${{ secrets.PULP_PASSWORD != '' }}" >> $GITHUB_OUTPUT
 
+    - name: Set environment variable to enable test cases requiring an enterprise license
+      if: steps.detect_if_should_run_enterprise.outputs.result == 'true'
+      id: set_run_enterprise_env
+      run: echo "KTF_TEST_RUN_ENTERPRISE_CASES=true" >> $GITHUB_ENV
```

Comment:
Is there a reason we need to introduce a new environment variable instead of using `KONG_LICENSE_DATA` https://github.com/Kong/kubernetes-testing-framework/blob/51dc612f3da531099d8ff3105d0018a9447b4c22/pkg/clusters/addons/kong/enterprise.go#L21 ?